apps/token-manager/app/src/web3-utils.js

```diff
@@ -1,4 +1,4 @@
-import { isAddress } from 'web3-utils'
+import { isAddress } from '@aragon/ui'
```

Comment:
[`isAddress()` from aragonUI](https://github.com/aragon/aragon-ui/blob/d668f801e8c3eea8ace21afdc4eeeaaedfaf4adf/src/utils/web3.js#L127-L137) is not as complete as [`isAddress()` from web3-utils](https://github.com/ethereum/web3.js/blob/1.x/packages/web3-utils/src/utils.js#L81-L99), but we are already using `js-sha3` in aragonUI so it should be easy to implement in a similar way.

We should ensure it passes these tests: https://github.com/ethereum/web3.js/blob/1.x/test/utils.isAddress.js